contracts/Counter.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+
+contract Counter {
```

Comment:
Q: what's the value of this contract?
For this repo to avoid it getting humongous we want contracts to either be showcasing Hedera functionality or EVM functionality. 
Goal is similar to the OZ repo, a showcase of functionality and paradigms.
This contract doesn't really do anything special. Is it possible to reuse any of the existing contracts?

If we merge this we'll likely move this later or replace it

Reply:
Agreed, this and its corresponding test contract can be removed